mpitests/test_apply.py

```diff
@@ -219,6 +219,86 @@ def circuit():
     assert np.allclose(local_state_vector, local_expected_output_cpu, atol=tol, rtol=0)
 
 
+def expval_single_wire_no_param(tol, obs):
+    num_wires = numQubits
+    comm = MPI.COMM_WORLD
+    commSize = comm.Get_size()
+    num_global_wires = commSize.bit_length() - 1
+    num_local_wires = num_wires - num_global_wires
+
+    state_vector = np.zeros(1 << num_wires).astype(np.complex128)
+    local_state_vector = np.zeros(1 << num_local_wires).astype(np.complex128)
+
+    state_vector = create_random_init_state(num_wires)
+    comm.Bcast(state_vector, root=0)
+
+    comm.Scatter(state_vector, local_state_vector, root=0)
+    dev_cpu = qml.device("default.qubit", wires=num_wires, c_dtype=np.complex128)
+
+    @qml.qnode(dev_cpu)
+    def circuit():
+        qml.QubitStateVector(state_vector, wires=range(num_wires))
+        return qml.expval(obs)
```

Comment:
Could avoid defining the circuit twice with something like
```suggestion
    def circuit():
        qml.QubitStateVector(state_vector, wires=range(num_wires))
        return qml.expval(obs)
    cpu_circuit = qml.qnode(circuit, dev_cpu)
    gpu_circuit = qml.qnode(circuit, dev_gpumpi)
```

Reply:
Good spot! Let me update it.

Reply:
Unfortunately, we can't implement in the way you mentioned above. Here is the error:  ```
cpu_circuit = qml.qnode(circuit, dev_cpu)
E       TypeError: <lambda>() takes 1 positional argument but 2 were given
```

Reply:
As discussed, updated with QNode(). Thanks